### DISCUSSION TITLE
Is seccomp-error-action=log supposed to do more than logging?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [netblue30/firejail#5896] [Q&A] [answered]
When I run firejail with seccomp-error-action, it seems that it changes seccomp? I'm testing the steam profile which has:
```
seccomp !chroot,!mount,!name_to_handle_at,!pivot_root,!ptrace,!umount2
seccomp.32 !process_vm_readv
```
by default. When I use the profile by default, vrchat's EAC doesn't load properly and Civilization VI says that I'm offline. If I comment these lines, they both work again. But when I wanted to debug a bit more with `--seccomp-error-action=log` and `journalctl --grep=SECCOMP -f`, it seems that it disables seccomp or something because it works again.

So my question is: is `seccomp-error-action=log` supposed to *just* do information logging?

**Comment by rusty-snake** [comment, marked as answer]
It enables audit mode, correct.
